docker-compose-dev.yaml

```diff
@@ -30,10 +31,10 @@ services:
 
   snapshotter-lite-v2:
     image: snapshotter-lite-v2
-    expose:
-      - 8002
-    ports:
-      - 8002:8002
+    # expose:
+    #   - 8002
+    # ports:
+    #   - 8002:8002
     volumes:
```

Comment:
I have commented out the core API port expose in build-dev for now. This change trickled down from the necessity to test the multi node setup issue. We can have a similar approach like I have with the `LOCAL_COLLECTOR_PORT` env var for the core API port bind and expose. Would like to know others' thoughts on this.

Reply:
Multi node setup issue for reference: https://github.com/PowerLoom/snapshotter-lite-multi-setup/issues/9

Reply:
I can think of two things here
1. We use just one snapshot collector for all the snapshotter lite nodes (not sure if that's possible architecture-wise), but this would be ideal IMO
2. Use a master node approach in a multi-node setup, i.e., only one node can expose core API, and others can keep it unexposed.

Reply:
1. I thought of that, but those are untested waters with regards to the capacity that can be supported. 

2. Including 1, this will basically require an additional compose file to be supported in case of specific ports to be bound and exposed for one kind of image, and no binding/expose for the other kind. 

These can be done, but has a high development overhead for a relatively small feature to be pushed out in the bigger picture, re: supporting multi node setup.

Reply:
I got it! My only concern is disabling port expose will make it difficult for developers or anyone running dev mode to check what's going on with Node (stats, data generated, etc) since they won't be able to access any core API endpoint.

We should add a flag that turns off port exposure in multi-node setup or think of something else.

Reply:
@xadahiya check commit https://github.com/PowerLoom/snapshotter-lite-v2/compare/bce84a7f80e6...9964aa05fba2

It exposes the core API on a custom env var if passed, else selects the default 8002